desktop/apps/consignments2/components/describe_work/index.js

```diff
@@ -84,134 +98,169 @@ function DescribeWork (props) {
+        <div className={b('row')}>
+          <div className={b('row-item')}>
+            <Field name='provenance' component={renderTextInput}
+              item={'provenance'}
+              instructions={'Where did you acquire this work?'}
+              label={'Provenance'}
+            />
+          </div>
         </div>
-      </div>
-      <div className={b('row')}>
-        <div className={b('row-item')}>
-          <div className={b('instructions')}>What city is the work located in?*</div>
-          <Autosuggest
-            suggestions={locationAutocompleteSuggestions}
-            onSuggestionsFetchRequested={fetchLocationSuggestionsAction}
-            onSuggestionsClearRequested={clearLocationSuggestionsAction}
-            onSuggestionSelected={chooseLocationAction}
-            getSuggestionValue={getSuggestionValue}
-            renderInputComponent={renderInputComponent}
-            renderSuggestion={renderSuggestion}
-            inputProps={locationAutosuggestInputProps}
-          />
+        <div className={b('row')}>
+          <div className={b('row-item')}>
+            <div className={b('instructions')}>What city is the work located in?*</div>
+            <div className={b('location-input', { disabled: locationAutocompleteFrozen })}>
+              <Autosuggest
+                suggestions={locationAutocompleteSuggestions}
+                onSuggestionsFetchRequested={fetchLocationSuggestionsAction}
+                onSuggestionsClearRequested={clearLocationSuggestionsAction}
+                onSuggestionSelected={chooseLocationAction}
+                getSuggestionValue={getSuggestionValue}
+                renderInputComponent={renderInputComponent}
+                renderSuggestion={renderSuggestion}
+                inputProps={locationAutosuggestInputProps}
+              />
+              { locationAutocompleteFrozen && <div className={b('unfreeze')} onClick={unfreezeLocationInputAction}><Close /></div> }
+            </div>
+          </div>
         </div>
-      </div>
-      <button
-        className={b('next-button').mix('avant-garde-button-black')}
-        disabled={pristine || invalid || locationAutocompleteValue.length === 0}
-        type='submit'
-      >
-        Next
-      </button>
-      {
-        error && <div className={b('error')}>{error}</div>
-      }
-    </form>
+        <button
+          className={b('next-button').mix('avant-garde-button-black')}
+          disabled={pristine || invalid || locationAutocompleteValue.length === 0 || !locationAutocompleteFrozen}
+          type='submit'
+        >
+          {
+            loading ? <div className='loading-spinner-white' /> : 'Submit'
+          }
+        </button>
+        {
+          error && <div className={b('error')}>{error}</div>
+        }
+      </form>
+    </div>
   )
 }
 
 const mapStateToProps = (state) => {
+  const selector = formValueSelector('describeWork')
+  const hasEditionValue = selector(state, 'edition')
```

Comment:
Since we're using `redux-form`, this is how we figure out whether or not the edition field has a value.